### PR TITLE
modified scroll behaviour of review activity

### LINKED ISSUE #2698: Improve peer review feature quality
**Summary:** 

After #2602 is merged #2253 is completed. Now some quality issues from hackathon still needs to be fixed.

- [ ] Code style issues @maskaravivek 
- [x] Move from XML to JSON @maskaravivek 
- [ ] It scrolls, I would prefer static/ support different screen sizes
- [x] Images moves on swipe, just questions should move instead @vanshikaarora 
- [ ] A beta alert should be added somewhere
- [x] Make image clickable - Open media details activity when clicked.
- [ ] Not showing the categories peer review screen for the images having zero categories.
- [ ] Or let the user categorize the image.
- [x] Adding a question mark next to "skip this image" button and Peer Review title which will display an explanation about their usage. @silkypriya 
- [x] Refactor asynctasks to Rxjava implementation @vanshikaarora 
- [ ] Write unit tests with kotlin @neslihanturan 
- [ ] Optimize randomizer
- [x] Extracting string resources for i18n

I am willing to work on some of them. But we can share, anyone wants to work on them please comment down.

### PR DESCRIPTION
**Pull request by vanshikaarora**
**Description (required)**
Fixes sub-task of #2698 [Scrolling behaviour of Review Activity]

### Changes made
Modified activity_review.xml and fragment_review_image.xml

**Screenshot**
WhatsApp Image 2019-03-24 at 1 43 19 PM

### CODE REVIEW

File: app/src/main/res/layout/activity_review.xml [hunk 1]
```diff
@@ -15,26 +15,29 @@
 
             <include layout="@layout/toolbar" />
 
+            <Button
+                android:id="@+id/skip_image"
+                android:layout_width="match_parent"
+                android:layout_height="25dp"
+                android:background="@android:color/transparent"
+                android:layout_below="@+id/toolbar"
+                android:text="SKIP THIS IMAGE"
```
Comment (domdomegg) [inline app/src/main/res/layout/activity_review.xml]: I know it wasn't previously, but it should use a `@string` resource rather than hardcoded text.

**Comment by vanshikaarora** [comment]
>I think it might be better if the whole page scrolled part from 'skip this image'. Currently just the bottom bit does.

@domdomegg before #2709 there was scrolling of whole activity instead of image, I removed whole activity scrolling in that PR .

In this current PR i adjusted the views to avoid scrolling every now and then. But it may scroll when occupies more than the extra space allotted. 

So should I revert the changes in scrolling that I did in #2709 ?

**Comment by domdomegg** [comment]
Sorry, I mean the whole activity should scroll up and down, and only the questions should scroll sideways - if that makes sense.

**Comment by vanshikaarora** [comment]
>I mean the whole activity should scroll up and down

Yes that's what happens previously. I'll revert the changes and send another PR :+1:

**Comment by domdomegg** [comment]
Before your previous PR (f7e6b20cab7a5819a1181bf98261453b2675dcdb) this is what used to happen:

untitled

The whole activity scrolls up and down, and the whole activity scrolls sideways

After your PR, only the bottom bit scrolls up and down, and only the bottom bit scrolls sideways

I meant to change it so that the whole activity scrolls up and down, and only the bottom bit scrolls sideways.

**Comment by vanshikaarora** [comment]
@domdomegg I have made the changes. Kindly review :)

**Comment by vanshikaarora** [comment]
Thanks for approving the PR @domdomegg :)

**Pull request merged**